src/main/java/datawave/microservice/query/logic/config/converter/StringToTypeConverter.java

```diff
@@ -0,0 +1,21 @@
+package datawave.microservice.query.logic.config.converter;
+
+import org.springframework.boot.context.properties.ConfigurationPropertiesBinding;
+import org.springframework.core.convert.converter.Converter;
+import org.springframework.stereotype.Component;
+
+import datawave.data.type.Type;
+
+@Component
+@ConfigurationPropertiesBinding
+public class StringToTypeConverter implements Converter<String,Type> {
+    @Override
+    public Type convert(String source) {
```

Comment:
Is there not method like this in a library? Not necessarily saying we use it, just seems like it would be code that is duplicated. Maybe thats ok.

Reply:
Hmmm, i'm not sure.  This is specifically used to map yaml spring properties to actual datawave Types, so im not sure that's something we've done before.  Looking in the type-utils project, I'm not seeing anything at first glance that could serve this purpose.